src/main/java/uk/gov/hmcts/et/common/model/ccd/items/TseAdminReplyTypeItem.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.et.common.model.ccd.items;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+import uk.gov.hmcts.et.common.model.ccd.types.TseAdminReplyType;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+@SuperBuilder
+@NoArgsConstructor
+public class TseAdminReplyTypeItem implements TseReplyTypeItem {
+
+    @JsonProperty("id")
+    private String id;
+    @JsonProperty("value")
+    private TseAdminReplyType value;
+
+    public String getDate() {
+        return this.value.getDate();
+    }
```

Comment:
Are these calls necessary? I feel like we should be keeping these objects as plain as possible and letting lombok deal with adding stuff. What if `value` on this object was null?

Reply:
good spot they are no longer used, I've removed them